MetaMask/metamask-mobile
# feat(ramp): offramp welcome screen copy updates

## **Description**

The welcome page for sell has different copy requirements. This PR updates copy based on `rampType`

## **Related issues**
N/A

## **Manual testing steps**
Check the sell version of the welcome page and verify that the copy has been updated

## **Screenshots/Recordings**

### **Before**
![Screenshot 2023-11-08 at 11 19 45 AM](https://github.com/MetaMask/metamask-mobile/assets/15269424/ad28160e-b334-40d2-bf10-d1347c9a4aa7)


### **After**
![Screenshot 2023-11-08 at 11 18 47 AM](https://github.com/MetaMask/metamask-mobile/assets/15269424/b085be31-e361-43ff-af8f-b8ac4341754c)


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
